### PR TITLE
Rails 5 compatibility

### LINKED ISSUE #2: Rails 5 compatibility
Have you checked it against Rails 5? After running the following:

```ruby
Feed.all.calculate_all(:visits_count)
```
I get: 
```ruby
fatal: exception reentered
from /home/vasilakisfil/.rvm/gems/ruby-2.3.1/gems/pry-0.10.4/lib/pry/pry_instance.rb:355:in `eval'
```

Feed is a relatively simple model.

### PR DESCRIPTION
**Pull request by StefSchenkelaars**
Here a pr for rails 5 compatibility. It works with groupdate 3 and rails 5. I did not check backward compatibility but all tests are green and I've added some comments in the `calculate_all` method, just for my own understanding but probably it will help others as well. Comments are welcome and it would be really appreciated if this can lead to a new release so I don't have to use my fork. 

It closes #2

### CODE REVIEW

**Comment by codesnik** [comment]
Sorry, I managed to not watch my own gem, missed your pull request.
I'll review your pull request shortly and will push update of the gem.
Thanks!

File: lib/calculate-all.rb [hunk 1]
```diff
@@ -1,89 +1,84 @@
...
     if functions.size == 1 && group_values.size == 0
       plain_rows = true
     end
 
+    # Final output hash
     results = {}
 
-    pluck(*group_values, *functions.values).each do |row|
+    # Fetch all the requested calculations from the database
+    # Note the map(&:to_s). It is required since groupdate returns a
```
Comment (codesnik) [inline lib/calculate-all.rb]: I'm not really sure that won't break some complex groupings.
But, hey, I didn't test for them, and I'm not an active user of my own library anymore (unlike, it seems, you), so I'm going to accept that pull request half-blindly, and will wait for a bug report, if any.

It also looks like gem still works with rails 4 and groupdate 2.

**Comment by codesnik** [comment]
A minor (or not) nitpick: changing coding style in irrelevant parts while fixing a particular problem makes reviewing harder. Sometimes it's a valid objection to reject a contribution.

I personally prefer double quotes everywhere, for a lot of reasons. But I wouldn't change quotes in someone else's library - blending in is more important.

But, then again, that library is small, and I'm not actively using it, so I'm going to accept that pull request.

Thanks for contribution!

**Comment by StefSchenkelaars** [comment]
Yea I should have split it up into multiple commits / pull requests but I only thought of that after finishing up everything. But thanks for the fast reply and release, now I can actually use it in my project 👍

**Comment by StefSchenkelaars** [comment]
And on the quotes: The reason to change it was that both single and double were used side by side, so I wanted to follow the rubocop style. But like I said before, I should have made that a separate pr.

**Pull request merged**